src/js/component.js

```diff
@@ -1553,22 +1572,22 @@ class Component {
     }
 
     if (name === 'Player' && Component.components_[name]) {
-      const Player = Component.components_[name];
+      const players = Component.components_[name].players;
```

Comment:
is it guaranteed that `Component.components_[name]` exists? I know when I added the check below it wasn't.